Description: Fix Neutron not installing stuff
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2013-07-30

--- neutron-2013.2~b2.orig/setup.cfg	2013-07-30 18:47:24.000000000 +0800
+++ neutron-2013.2~b2/setup.cfg	2013-07-30 18:53:36.000000000 +0800
@@ -133,3 +133,8 @@
 domain = neutron
 output_dir = neutron/locale
 input_file = neutron/locale/neutron.pot
+
+[egg_info]
+tag_build = 
+tag_date = 0
+tag_svn_revision = 0
