Description: Fix miagration2
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: http://bugs.debian.org/726719
Last-Update: 2013-10-19

--- neutron-2013.2.orig/neutron/db/migration/alembic_migrations/versions/128e042a2b68_ext_gw_mode.py
+++ neutron-2013.2/neutron/db/migration/alembic_migrations/versions/128e042a2b68_ext_gw_mode.py
@@ -54,7 +54,7 @@ def upgrade(active_plugins=None, options
     op.add_column('routers', sa.Column('enable_snat', sa.Boolean(),
                                        nullable=False, server_default='1'))
     # Set enable_snat to True for existing routers
-    op.execute("UPDATE routers SET enable_snat=True")
+    op.execute("UPDATE routers SET enable_snat='True'")
 
 
 def downgrade(active_plugins=None, options=None):
