Description: Fix a valid default for interface_driver
Author: Thomas Goirand <zigo@debian.org>
Last-Update: 2013-09-20

--- neutron-2013.2~b3.orig/etc/l3_agent.ini
+++ neutron-2013.2~b3/etc/l3_agent.ini
@@ -8,7 +8,7 @@
 
 # Example of interface_driver option for OVS based plugins (OVS, Ryu, NEC)
 # that supports L3 agent
-# interface_driver = neutron.agent.linux.interface.OVSInterfaceDriver
+interface_driver = neutron.agent.linux.interface.OVSInterfaceDriver
 
 # Use veth for an OVS interface or not.
 # Support kernels with limited namespace support
--- neutron-2013.2~b3.orig/etc/lbaas_agent.ini
+++ neutron-2013.2~b3/etc/lbaas_agent.ini
@@ -13,7 +13,7 @@
 
 # Example of interface_driver option for OVS based plugins (OVS, Ryu, NEC, NVP,
 # BigSwitch/Floodlight)
-# interface_driver = neutron.agent.linux.interface.OVSInterfaceDriver
+interface_driver = neutron.agent.linux.interface.OVSInterfaceDriver
 
 # Use veth for an OVS interface or not.
 # Support kernels with limited namespace support
