Description: Use proper argument in add_column method call
Author: Eugene Nikanorov <enikanorov@mirantis.com>
Origin: upstream, https://review.openstack.org/#/c/52636/
Bug-Debian: http://bugs.debian.org/726719
Bug-Ubuntu: https://launchpad.net/bugs/1241577
Last-Update: Fri, 18 Oct 2013 21:50:42 +0800 (17:50 +0400)

--- neutron-2013.2.orig/neutron/db/migration/alembic_migrations/versions/128e042a2b68_ext_gw_mode.py
+++ neutron-2013.2/neutron/db/migration/alembic_migrations/versions/128e042a2b68_ext_gw_mode.py
@@ -52,7 +52,7 @@ def upgrade(active_plugins=None, options
         return
 
     op.add_column('routers', sa.Column('enable_snat', sa.Boolean(),
-                                       nullable=False, default=True))
+                                       nullable=False, server_default=True))
     # Set enable_snat to True for existing routers
     op.execute("UPDATE routers SET enable_snat=True")
 
